model_templates/python_multiple_codegen/README.md

```diff
@@ -0,0 +1,16 @@
+## Python + Multiple CodeGen models
+
+### Build java environment
+`cd ./datarobot-user-models/public_dropin_environments/java_codegen`   
+`docker build -t java_env .`
+
+Then run locally
+### To run locally using 'drum'
+Paths are relative to `./datarobot-user-models`:   
```

Comment:
Maybe this is just me, but `.` in the beginning confuses me.
I'd say "Paths are relative to the repository root (`datarobot-user-models`)"

Reply:
DONE